README.md

```diff
@@ -29,14 +29,28 @@ For an existing react application, follow the steps bellow:
 
 1. Load the fiori-fundamentals styles. If using create-react-app, this will be in `App.css`.
 
-    ```
+    ```scss
     @import '~fiori-fundamentals/dist/fiori-fundamentals.min.css';
     ```
 
 1. Import components as needed. See [Component Documentation](https://sap.github.io/fundamental-react/) for examples and API details.
-    ```
-    import { Alert } from 'fundamental-react';
-    ```
+```javascript
+import { Alert } from 'fundamental-react/lib/Alert/Alert';
```

Comment:
Shouldn't `import Alert from 'fundamental-react/lib/Alert/Alert'` also work in this case?

Reply:
@jeffredodd As of now anyway, there are no default exports in any of the source modules so named exports are all we have to work with.

Reply:
I don't think so since they aren't default exports.